pallets/loans/src/math.rs

```diff
@@ -145,8 +145,8 @@ pub(crate) fn valid_write_off_group<Rate>(
 	maturity_date: Moment,
 	now: Moment,
 	write_off_groups: &[WriteOffGroup<Rate>],
-) -> Result<Option<u32>, DispatchError> {
-	let mut index = None;
+) -> Result<Option<(u32, &WriteOffGroup<Rate>)>, DispatchError> {
```

Comment:
Could we have a type alias for `(u32, &WriteOffGroup<Rate>)`? Like `(Index, &WriteOffGroup<Rate>)` or a descriptive alias for the tuple itself.